packages/bls/src/index.ts

```diff
@@ -1,15 +1,18 @@
 import {Keypair} from "./keypair";
 import {PrivateKey} from "./privateKey";
-import {G2point} from "./helpers/g2point";
-import {G1point} from "./helpers/g1point";
 import {PublicKey} from "./publicKey";
 import {Signature} from "./signature";
-import {ElipticCurvePairing} from "./helpers/ec-pairing";
-import ctx from "./ctx";
 import {BLSPubkey, BLSSecretKey, BLSSignature, Domain, Hash} from "@chainsafe/eth2.0-types";
+import {init} from "./context";
+import {PUBLIC_KEY_LENGTH} from "./constants";
 
 export {Keypair, PrivateKey, PublicKey, Signature};
 
+
+export async function initLibrary(): Promise<void> {
```

Comment:
I need to rename one of them or IDE is going wild with typehinting both of them haha

Reply:
What do you think of renaming to `initBLS`. Might be better name, because whomever is using this library would have to rename that export every time so he is aware what's going on. Random `await init()` in your code is confusing

Reply:
> What do you think of renaming to initBLS

sgtm :)